starboard/nplb/BUILD.gn

```diff
@@ -297,4 +311,7 @@ target(gtest_target_type, "nplb") {
       "//starboard/shared/starboard/player:video_dmp",
     ]
   }
+  if (is_linux) {
+    sources += [ "//starboard/linux/x64x11/main.cc" ]
```

Comment:
TODO @andrewsavage1 see if we can use starboard_platform_with_main (before it had duplicate symbols)